src/main/java/uk/ac/sanger/sccp/stan/service/InPlaceOpCommentServiceImp.java

```diff
@@ -121,7 +121,7 @@ public Map<Integer, Comment> loadComments(Collection<String> problems, Collectio
         var comments = commentValidationService.validateCommentIds(problems,
                 barcodesAndCommentIds.stream().map(BarcodeAndCommentId::getCommentId));
         return comments.stream()
-                .collect(BasicUtils.toMap(Comment::getId));
+                .collect(BasicUtils.inMap(Comment::getId));
```

Comment:
What are these 'toMap' to 'inMap' changes?

Reply:
I renamed the method in BasicUtils to avoid confusing it with the built-in Collectors.toMap method.